### ISSUE TITLE
Consider adding `require-atomic-updates` when it becomes more stable

### ISSUE THREAD

**Issue by manovotny** [manovotny/eslint-config-get-off-my-lawn#112]
As much as we want to include it, it's just too unstable right now and causes too many false positives right now.

* https://github.com/eslint/eslint/issues/11899
* https://github.com/eslint/eslint/issues/11954
* https://github.com/eslint/eslint/issues/11900

**Comment by roboli** [comment]
It won't let me do this for example

    `let action;

    const withAction = async () => {
      if(!action) {
        action = await actions.getOne({ id });
      }
      return { action };
    };`

Do you think is also due the above? Thanks.

**Comment by manovotny** [comment]
@roboli yep, that's exactly why I removed it from v5 after beta feedback.

I just released `5.0.0-beta.5`, which no longer contains the rule.

**Comment by manovotny** [comment]
Updated original message. Two of three issues fixed! Progress, but still holding off on adoption for now.
